src/url.js

```diff
@@ -67,6 +71,21 @@ export function parseUrl(url) {
     search: a.search,
     hash: a.hash,
   };
+
+  // Some IE11 specific polyfills.
+  // 1) IE11 strips out the leading '/' in the pathname.
+  if (info.pathname === '' || info.pathname[0] !== '/') {
```

Comment:
We should be able to drop the first conditional, since the second implies the first.